src/AutoMerge/public.jl

```diff
@@ -1,5 +1,64 @@
 """
-    run(env, cicfg::CIService; kwargs...)
+    run([env, cicfg::CIService]; kwargs...)
+
+Run the `RegistryCI.AutoMerge` service.
+
+# Arguments
+
+- `env`:
+- `ciccfg`:
+
+# Keyword Arguments
+
+- `merge_new_packages`:
+- `merge_new_versions`:
+- `new_package_waiting_period`: new package waiting period, e.g `Day(3)`.
+- `new_jll_package_waiting_period`: new JLL package waiting period, e.g `Minute(20)`.
+- `new_version_waiting_period`: new package version waiting period, e.g `Minute(10)`.
+- `new_jll_version_waiting_period`: new JLL package version waiting period, e.g `Minute(10)`.
+- `registry`: the registry name you want to run AutoMerge on.
+- `tagbot_enabled`: if tagbot is enabled.
+- `authorized_authors`: list of who can submit registration, e.g `String["JuliaRegistrator"]`.
+- `authorized_authors_special_jll_exceptions`:
+- `additional_statuses`:
+- `additional_check_runs`:
+- `error_exit_if_automerge_not_applicable`:
```

Comment:
not totally sure about these, probably @DilumAluthge knows

Reply:
I went ahead and documented them. @ericphanson take a look, and resolve this conversation if the explanations look right.